src/backend/vulkan/src/physical_device.rs

```diff
@@ -1083,6 +1156,63 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
         }
     }
 
+    fn query_external_buffer_properties(
+        &self,
+        usage: hal::buffer::Usage,
+        sparse: hal::memory::SparseFlags,
+        memory_type: hal::external_memory::ExternalMemoryType,
+    ) -> Result<
+        hal::external_memory::ExternalBufferProperties,
+        hal::external_memory::ExternalMemoryQueryError,
+    > {
+        let external_memory_capabilities_extension = match &self
```

Comment:
Let's not match on a reference. Instead, do ref binding, i.e. `Some(ref functor)`

Reply:
Done!